test/openapi_generator_test.py

```diff
@@ -2080,25 +2080,26 @@ def void_post(self, unused_request):
       def void_post_audience(self, unused_request):
         return message_types.VoidMessage()
 
+    with pytest.raises(api_exceptions.ApiConfigurationError):
+      self.generator.pretty_print_config_to_json(Auth0Service)
+
     @api_config.api(name='root', hostname='example.appspot.com',
-                    version='v1', issuers=ISSUERS, audiences=['auth0audapi'])
-    class MyService2(remote.Service):
+                    version='v1')
+    class ApiKeyAndGoogleAuthService(remote.Service):
       """Describes MyService."""
 
       @api_config.method(message_types.VoidMessage, message_types.VoidMessage, path='void',
-                         http_method='POST', name='void')
+                         http_method='POST', name='void', api_key_required=True)
       def void_post(self, unused_request):
         return message_types.VoidMessage()
 
       @api_config.method(message_types.VoidMessage, message_types.VoidMessage, path='void3',
-                         http_method='POST', name='void3', audiences=['auth0audmethod'])
+                         http_method='POST', name='void3', audiences=['google-auth-audience'])
       def void_post_audience(self, unused_request):
         return message_types.VoidMessage()
 
-    with pytest.raises(api_exceptions.ApiConfigurationError):
-      self.generator.pretty_print_config_to_json(MyService1)
-    with pytest.raises(api_exceptions.ApiConfigurationError):
-      self.generator.pretty_print_config_to_json(MyService2)
```

Comment:
was the MyService2 exception test previously verifying something else that the MyService1 test wasn't?

Reply:
It was not; the two services were identical. I don't remember why it was in there